protocol-definitions/Jet.yaml

```diff
@@ -516,3 +516,28 @@ methods:
           since: 2.6
           doc: |
             True if registration is removed, false otherwise.
+  - id: 21
+    name: updateJobConfig
+    since: 2.6
+    doc: ''
+    request:
+      retryable: true
+      partitionIdentifier: -1
+      params:
+        - name: jobId
+          type: long
+          nullable: false
+          since: 2.6
+          doc: ''
+        - name: deltaConfig
+          type: Data
```

Comment:
see https://github.com/hazelcast/hazelcast-client-protocol/pull/450#discussion_r1095625250 and https://github.com/hazelcast/hazelcast-client-protocol/pull/450#discussion_r1127671986

Reply:
@ihsandemir can you take a look at these comments? they don't require backward compatibility for Jet. I would like to merge this PR ASAP as the Hazelcast PR is merged.

Reply:
I see that backward compatibility is not even an option for Jet and I am not happy about that for our clients since we propose a very strong compatibility guarantee in all other APIs and having this exception case for Jet is really not good for users.